internal/server/target_test.go

```diff
@@ -540,6 +540,66 @@ func TestTarget_EnforceMaxBodySizes(t *testing.T) {
 	})
 }
 
+func TestTarget_buildHealthCheckURL(t *testing.T) {
+	tests := []struct {
+		name            string
+		targetURL       string
+		healthCheckPort int
+		healthCheckPath string
+		expectedURL     string
+	}{
+		{
+			name:            "default port - same as target",
+			targetURL:       "localhost:3000",
+			healthCheckPort: 0,
```

Comment:
Is `--health-check-port` still optional? I noticed your checking if it's > 0 but most people won't be specifying it at the moment.

Reply:
thanks, replied in other thread